apps/indexer/lib/indexer/supervisor.ex

```diff
@@ -198,8 +196,19 @@ defmodule Indexer.Supervisor do
         fetchers_with_amb_bridge_mediators
       end
 
+    fetcher_with_epoch_rewards =
+      if System.get_env("DISPLAY_REWARDS") === "true" do
```

Comment:
Why System instead of Application here?

Reply:
same story as https://github.com/celo-org/blockscout/pull/705#discussion_r928673766 ;-)